Update documentation for locale configurations (#1159)
DetachHead/basedpyright: docs/configuration/config-files.md

## Locale Configuration

> Pyright provides diagnostic messages that are translated to multiple languages, which are also introduced in basedpyright but with [community-contributed translations](../development/localization.md) in a few languages. By default, basedpyright uses the default locale of the operating system. You can override the desired locale through the use of one of the following environment variables, listed in priority order.

Reviewer: ```suggestion
Pyright provides diagnostic messages that are translated to multiple languages, which are improved in basedpyright thanks to [community-contributed translations](../development/localization.md) for a few languages. By default, basedpyright uses the default locale of the operating system. You can override the desired locale through the use of one of the following environment variables, listed in priority order.
```

```
LC_ALL="de"
LC_MESSAGES="en-us"
LANG="zh_CN"
LANGUAGE="fr"
```

> The locale specifiers can be `xx-xx` or `xx_XX` in basedpyright. The latter form is used in unix-like platforms, while which is not supported in pyright.

Reviewer: ```suggestion
The locale specifiers can be `xx-xx` or `xx_XX` in basedpyright. The latter form is used in unix-like platforms, which is not supported in pyright.
```
